Makefile

```
@@ -213,6 +214,28 @@ $(BUILD_DIR)/darwin-amd64/lifecycle/launcher:
 	$(GOENV) $(GOBUILD) -o $(OUT_DIR)/launcher -a ./cmd/launcher
 	test $$(du -m $(OUT_DIR)/launcher|cut -f 1) -le 4
 
+generate-sbom: build-sbom-generator-image generate-sbom-lifecycle generate-sbom-launcher copy-sboms
+
+copy-sboms: export GOOS:=linux
+copy-sboms: export GOARCH:=amd64
+copy-sboms: OUT_DIR:=$(BUILD_DIR)/$(GOOS)-$(GOARCH)/lifecycle
+copy-sboms:
+	@echo "> Copying SBOMs"
+	cp $(OUT_DIR)/*.json $(PWD)/out/linux-arm64/lifecycle
+	cp $(OUT_DIR)/*.json $(PWD)/out/windows-amd64/lifecycle
+
+build-sbom-generator-image:
+	@echo "> Building docker image for sbom generator"
+	DOCKER_BUILDKIT=1 docker build --progress=plain --file tools/sboms/Dockerfile --tag $(SBOM_GENERATOR_IMAGE) --cache-from=$(SBOM_GENERATOR_IMAGE) .
```

```suggestion
	@echo "> Building docker image for sbom generator"
	DOCKER_BUILDKIT=1 docker build --progress=plain --file tools/sboms/Dockerfile --tag $(SBOM_GENERATOR_IMAGE) .
```
I could be mistaken but I don't think this is needed if we're also building to `SBOM_GENERATOR_IMAGE`

---

ok. I'll remove the parameter.